### DISCUSSION TITLE
Upgrade fails

### DISCUSSION THREAD

**Discussion by Wamor** [Koenkk/zigbee2mqtt#17738] [General]
Hello, my name is Wamor and I am using zigbee2mqtt for a long time. Today I have received my new Lidl switch adapter and they where not recognized by the current version I was using. Therefore I upgraded to the latest zigbee2mqtt version without success.
Starting zigbee2mqtt (npm start in /opt/zigbee2mqtt) gives me the following error.
> zigbee2mqtt@1.30.4-dev start /opt/zigbee2mqtt
> node index.js

/opt/zigbee2mqtt/node_modules/rimraf/node_modules/minimatch/dist/cjs/ast.js:86
    #fillNegs() {
             ^

SyntaxError: Unexpected token '('

Does anybody have a clue what is going wrong here?
Thank you in advance for helping me.

Regards,

Wamor

**Comment by ghost** [comment]
This is also happening to me, were you able to solve it?

**Reply by ghost** [reply]
Okay so I fixed it. Basically I was using node version `12.22.12` and I have upgraded it to latest `18.16.0`. Also just in case I upgraded npm from `9.5.1` to `9.7.1`. Now it is working fine
